dashboard/config/blocks/GamelabJr/gamelab_makeNewScaledSprite.js

```diff
@@ -0,0 +1,33 @@
+function makeNewScaledSprite(costume,location) {
+  createNewSprite({name: 'temporarySprite'}, costume, location);
+  var size;
+  switch (costume) {
+    case 'wolf':
+    case 'wolf_sunglasses':
+      size = 208 * 8 / 7;
```

Comment:
Why `8 / 7`?

Reply:
This file has been replaced with new ones found here: 
https://github.com/code-dot-org/code-dot-org/tree/staging/dashboard/config/blocks/BadGuys

The calculations there were just done super quickly to get us to some kind of working proof of concept. Each sprite now has a default size value based on its costume and can be automatically scaled while keeping proportions true to what the partner provided.